Sources/RichTextKit/Context/RichTextContext.swift

```diff
@@ -55,114 +55,91 @@ public class RichTextContext: ObservableObject {
 
     /// Whether or not the indent level can be increased.
     @Published
-    public var canIncreaseIndent = true
-
-    /// The current font name.
-    @Published
-    public var fontName = ""
-
-    /// The current font size.
-    @Published
-    public var fontSize = CGFloat.standardRichTextFontSize
-
+    var canIncreaseIndent = true
+    
     /// The current foreground color, if any.
     @Published
-    public var foregroundColor: ColorRepresentable?
-
-    /// The currently highlighted range, if any.
-    @Published
-    public var highlightedRange: NSRange?
+    var foregroundColor: ColorRepresentable?
 
     /// The style to apply when highlighting a range.
     @Published
-    public var highlightingStyle = RichTextHighlightingStyle.standard
+    var highlightingStyle = RichTextHighlightingStyle.standard
 
     /// Whether or not the current text is bold.
     @Published
-    public var isBold = false
-
-    /// Whether or not the text is currently being edited.
-    @Published
-    public var isEditingText = false
+    var isBold = false
 
     /// Whether or not the current text is italic.
     @Published
-    public var isItalic = false
+    var isItalic = false
 
     /// Whether or not the current text is striked through.
     @Published
-    public var isStrikethrough = false
+    var isStrikethrough = false
 
     /// Whether or not the current text is underlined.
     @Published
-    public var isUnderlined = false
+    var isUnderlined = false
 
     /// The current strikethrough color, if any.
     @Published
-    public var strikethroughColor: ColorRepresentable?
+    var strikethroughColor: ColorRepresentable?
 
     /// The current stroke color, if any.
     @Published
-    public var strokeColor: ColorRepresentable?
-
-    /// The current text alignment, if any.
-    @Published
-    public var textAlignment: RichTextAlignment = .left
+    var strokeColor: ColorRepresentable?
 
     /// The current underline color, if any.
     @Published
-    public var underlineColor: ColorRepresentable?
-
-
-    // MARK: - Internal trigger properties
-
-    /// Set this property to trigger a certain action.
-    @Published
-    var triggerAction: RichTextAction?
-
-    /// Set this property to trigger a paste operation.
-    @Published
-    var shouldPasteImage: (image: ImageRepresentable, atIndex: Int, moveCursor: Bool)?
-
-    /// Set this property to trigger an image paste operation.
-    @Published
-    var shouldPasteImages: (images: [ImageRepresentable], atIndex: Int, moveCursor: Bool)?
-
-    /// Set this property to trigger a text paste operation.
-    @Published
-    var shouldPasteText: (text: String, atIndex: Int, moveCursor: Bool)?
-
-    /// Set this property to trigger a string update.
-    @Published
-    var shouldSetAttributedString: NSAttributedString?
-
-    /// Set this property to trigger a range change.
-    @Published
-    var shouldSelectRange = NSRange()
+    var underlineColor: ColorRepresentable?
```

Comment:
Perhaps we should have a `colors[RichTextColor: ColorRepresentable]` dictionary instead?

Reply:
Hmm next PR maybe? Or keep it this way until now? I dont have strong opinion, maybe I am more headed towards keeping as is 😅